.github/ISSUE_TEMPLATE/changeoption.yml

```
@@ -0,0 +1,32 @@
+---
+description: Change an existing target.
+labels: [change-target]
+name: Change target
+title: "<TARGET>: <DESCRIPTION>"
+
+body:
+  - type: checkboxes
+
+    attributes:
+      label: Has this change been requested before?
+      description: >-
+        Search [existing issues](https://github.com/danth/stylix/issues) to
+        ensure that this change hasn't been requested before.
+
+    options:
+      - label: >-
+          I have searched for and not found existing issues requesting this
+          change.
+        required: true
+
+  - type: textarea
+
+    attributes:
+      label: Description
+      description: >-
+        Describe the change. Consider including how this will affect existing
+        and future configurations and why this behavior will be better than
+        the current behavior.
```

What about this:

```suggestion
        Describe the change and its potential breaking changes, and demonstrate
        the use case it improves.
```

This could probably be further improved, but I am unsure to what extend now.

---

> This could probably be further improved, but I am unsure to what extend now.

I think this looks good. As much as we want to give guidance, it's hard to be general enough for every possible change someone could be requesting. We can iterate later based on how people end up using the section.